frontend/src/app/Settings/Users/UserDetailUtils.tsx

```diff
@@ -0,0 +1,201 @@
+import React, { useState } from "react";
+
+import { displayFullName } from "../../utils";
+import Button from "../../commonComponents/Button/Button";
+import { capitalizeText, formatUserStatus } from "../../utils/text";
+import { ReactComponent as DeactivatedIcon } from "../../../img/account-deactivated.svg";
+import { ReactComponent as PendingIcon } from "../../../img/account-pending.svg";
+import { OktaUserStatus } from "../../utils/user";
+import Alert from "../../commonComponents/Alert";
+
+import { SettingsUser } from "./ManageUsersContainer";
+import ReactivateUserModal from "./ReactivateUserModal";
+import ResendActivationEmailModal from "./ResendActivationEmailModal";
+import "./ManageUsers.scss";
+
+export const UserStatusSubheading: React.FC<{ user: SettingsUser }> = ({
+  user,
+}) => {
+  function getUserStatusText() {
```

Comment:
Did we also want to handle the `RECOVERY` status as well? I remember in [this doc](https://docs.google.com/document/d/1_XAuO35CC3e0aAAaCe1dk2BfPl_agztMBafT1E9GWiw/edit) we said it would be helpful to surface this but I forget what conclusion we made. 😅 
<img width="402" alt="Screenshot 2023-08-16 at 10 28 07" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/52ba244f-e698-4914-babb-9bfd97c91119">

Reply:
I think the group got a consensus that the delay status was a nice to have but extra work that should be done later (specially to not delay this implementation). However, I don't think we ever created a ticket to track it at all. Tagging @mehansen for instructions on how to move forward!.